packages/react/alert-dialog/src/AlertDialog.stories.tsx

```diff
@@ -99,6 +99,18 @@ export const Chromatic = () => (
           <AlertDialogCancel className={cancelClass}>Cancel</AlertDialogCancel>
         </AlertDialogContent>
       </AlertDialog>
+
+      <h2>Open with reordered parts</h2>
+      <AlertDialog defaultOpen>
+        <AlertDialogOverlay className={overlayClass} style={{ right: '50%', bottom: '50%' }} />
+        <AlertDialogContent className={chromaticContentClass} style={{ top: '25%', left: '25%' }}>
+          <AlertDialogTitle className={titleClass}>Title</AlertDialogTitle>
+          <AlertDialogDescription className={descriptionClass}>Description</AlertDialogDescription>
+          <AlertDialogAction className={actionClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+        <AlertDialogTrigger className={triggerClass}>delete everything</AlertDialogTrigger>
+      </AlertDialog>
```

Comment:
I added these stories to all of the `Trigger` components because I thought it made sense that we would test this works for all in-case ref stuff changes internally.